dbtmetabase/metabase.py

```diff
@@ -1,16 +1,26 @@
+import json
 import logging
+from typing import Any, Sequence, Optional, Tuple, Iterable, MutableMapping
+
 import requests
-import json
 import time
-from typing import Any, Optional, Tuple, Dict
+
+from .models.metabase import MetabaseModel, MetabaseColumn
 
 
 class MetabaseClient:
     """Metabase API client."""
 
     _SYNC_PERIOD_SECS = 5
 
-    def __init__(self, host: str, user: str, password: str, https=True):
+    def __init__(
+        self,
+        host: str,
+        user: str,
+        password: str,
+        https: bool = True,
+        verify: Any = None,
```

Comment:
Isn't `verify` a `str` at this point?

Reply:
I went ahead and made this more explicit to reflect that it should be either bool or str